pkg/controller/wildflyserver/wildflyserver_controller.go

```diff
@@ -139,7 +139,7 @@ func (r *ReconcileWildFlyServer) Reconcile(request reconcile.Request) (reconcile
 	}
 
 	if ok, err := validate(wildflyServer); !ok {
-		return r.ManageError(wildflyServer, err)
+		return r.manageError(wildflyServer, err)
```

Comment:
Thanks for this. The idea was to use this method from the outside by using a validation webhook, but sounds fine to reduce the scope now.

Reply:
Oh, I didn't know the context that the method should be used outside. I just checked where it's used and it made sense to me make it private. If you say I will change it back.

Reply:
No, it is fine as it is, thanks!

Reply:
Thanks.